src/main/java/org/reaktivity/nukleus/http_cache/internal/proxy/cache/DefaultCacheEntry.java

```diff
@@ -55,6 +55,7 @@
 
     private final DefaultCache cache;
     private final int requestHash;
+    private final int collectionHash;
```

Comment:
This should be renamed to `requestHashWithoutQuery`, right?

Reply:
yep